rust/Cargo.toml

```diff
@@ -11,29 +11,21 @@ build = "build.rs"
 failure = "0.1"
 ffi_helpers = "0.2"
 hex = "0.4"
-zcash_client_backend = "0.5"
-zcash_client_sqlite = "0.3"
+zcash_client_backend = { version = "0.5", features = ["transparent-inputs"] }
+zcash_client_sqlite = { version = "0.3", features = ["transparent-inputs"] }
 zcash_primitives = "0.5"
 zcash_proofs = "0.5"
-
+orchard = "=0.1.0-beta.3"
 #### Temporary additions: ####################################
-#bitvec = "0.22"
 base58 = "0.1.0"
-#sha2 = "0.9"
-#bs58 = { version = "0.3", features = ["check"] }
-#hdwallet = "0.3.0"
-#ripemd160 = "0.9"
-hdwallet = { git = "https://github.com/nuttycom/hdwallet", rev = "72f1f7a56c114eed484cefd6d402b7ef28158712"}
-secp256k1 = "0.19"
+sha2 = "0.9"
+hdwallet = "0.3.0"
+ripemd160 = "0.9"
```

Comment:
The [`ripemd160` crate](https://crates.io/crates/ripemd160) is deprecated. Zcashd switched to using `ripemd` instead.